src/components/SignInModal.tsx

```diff
@@ -172,6 +235,22 @@ export const SignInModal: React.FC<SignInModalProps> = ({
                     </p>
                   </div>
                 )}
+                {isLinking && (
+                  <div>
+                    <p className="text-red-700 dark:text-red-300">
+                      An account with this email already exists, please sign in
+                      with Google to link the two accounts:
```

Comment:
maybe clear this when the modal is closed? could be annoying if you accidentally close the modal, but right now the message persists if you sign in without linking, which looks sort of buggy

https://user-images.githubusercontent.com/63619830/155862286-cbc5a1e2-557f-4d41-906b-e9899cb7eabc.mov